docs/spark-connector/spark-catalog-jdbc.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Spark connector JDBC catalog"
+slug: /spark-connector/spark-catalog-jdbc
+keyword: spark connector jdbc catalog
+license: "This software is licensed under the Apache License version 2."
+---
+
+The Apache Gravitino Spark connector offers the capability to read JDBC tables, with the metadata managed by the Gravitino server. To enable the use of the JDBC catalog within the Spark connector, you must download the jdbc driver jar which you used to Spark classpath.
```

Comment:
please add an index in `spark-connector.md`,  https://github.com/apache/gravitino/blob/main/docs/spark-connector/spark-connector.md?plain=1?#L14